src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -67,6 +67,7 @@ const VideoDetailPage = () => {
         video: videoData?.videoUrl,
         courseKey: videoData?.courseKey,
         title: videoData?.courseTitle,
+        video_usage_key: videoData?.videoUsageKey,
```

Comment:
[nit] Should the key for this value be camelCased as `videoUsageKey` to keep in parity with `courseKey`

Reply:
I dunno... maybe. I made it snake case as this is how it's formatted in the column name I'll join against in the database, which is the only place this name will be seen. I made a choice here to optimize for the analysis. 

If this variable was used again in the JS I would 100% change it.